packages/maskbook/src/plugins/Gitcoin/base.ts

```diff
@@ -0,0 +1,15 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { PLUGIN_ID, PLUGIN_ICON, PLUGIN_NAME, PLUGIN_DESCRIPTION } from './constants'
+
+export const base: Plugin.Shared.Definition = {
+    ID: PLUGIN_ID,
+    icon: PLUGIN_ICON,
+    name: { fallback: PLUGIN_NAME },
+    description: { fallback: PLUGIN_DESCRIPTION },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
+    enableRequirement: {
+        architecture: { app: false, web: true },
+        networks: { type: 'opt-out', networks: {} },
+        target: 'insider',
```

Comment:
```suggestion
        target: 'stable',
```

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/3495/commits/804dac51f21d5506d29f8f214d703420b6b3816a